la.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+<head>
+<title>Our Funky HTML Page</title>
+<link rel="stylesheet" type="text/css" href="./vlog.css"/>
+<meta name="description" content="Our first page">
+<meta name="keywords" content="html tutorial template">
+</head>
+<body>
+
+<div class="main-menu">
+    <div class="main-menu_top">
+        <div class="logo"> 
+            <img class="big_logo" src="linklogo.png">
+        <div class="headline"> <h1>Jack</h1> </div>
+        <div class="small_headline"> Web Developer </div>
+
+        </div>
+
+    </div>
+      <div class="vertical-menu">
+        <a href="#" class="About">About</a>
+        <a href="#" class="My_Skills">My Skills</a>
+        <a href="#" class="Work">Work</a>
+        <a href="#" class="Contact">Contact</a>
+        <a href="#" class="Blog">Blog</a>
+      </div>
+
+      <div class="social_media">
```

Comment:
do not use slash unless it's a element or modification, see BEM :D

Reply:
Done